### ISSUE TITLE
Missing rice.rb in release

### ISSUE THREAD

**Issue by mvastola** [jasonroelofs/rice#188]
Hi, so I'm not sure why, but there seems to be an issue with the published gem (forcing me to use git): `lib/rice.rb` seems to be missing from the distributed version installed by `gem`/`bundle`.

Just in case you weren't aware.

```bash
mvastola@vastdesk:~/tmp$ gem unpack rice && ls -l rice-4.1.0/lib/ && rm -Rf rice-4.1.0
Unpacked gem: '/tmp/mvastola/rice-4.1.0'
total 12
-rw-r--r-- 1 mvastola mvastola 4322 Aug 13 20:47 mkmf-rice.rb
-rw-r--r-- 1 mvastola mvastola   36 Aug 13 20:47 version.rb
```

**Comment by jasonroelofs** [comment]
What error are you seeing? With the 4.x release that turned this into a header-only library, there's really no need to `require 'rice'` (it doesn't do anything anyway).

**Comment by mvastola** [comment]
Oops. I was seeing `cannot load such file` upon `bundle exec rake compile` of my in-progress gem. I ended up with it because I used https://github.com/jasonroelofs/rice/blob/0c4199bbbea3344faf7ad04f70edb55c38613dd7/sample/callbacks/extconf.rb as my starter code.

I just assumed the file was needed, so I looked for how I could ensure it existed, so I set the source to git. Now that you point it out though, it should have clicked that because the file was empty I could just remove the `require` (so I'll do that). I saw the comment `required to load the Rice gem` and I didn't question it.
